### ISSUE TITLE
Function call appended to constructor breaks hover and signature help on constructor

### ISSUE THREAD

**Issue by ylazy** [BowlerHatLLC/vscode-as3mxml#657]
Hi!

```as
var test:ChainTest = new ChainTest("foo", "bar").chain(1, 2);
```

When you type `,` after `"bar"`, code hint doesn't show.
When you roll over to `ChainTest` in `new ChainTest`, code description doesn't show.

class ChainTest:

```as
package 
{
	public class ChainTest
	{
		public function ChainTest(param1:String = null, param2:String = null, param3:String = null):void
		{
			
		}
		
		public function chain(param1:int = 0, param2:int = 0):ChainTest
		{
			return this;
		}
		
	}
}
```

Thanks!

**Comment by joshtynjala** [comment]
I've determined that this is a bug in the Royale AS3 compiler, which is used for code intelligence in vscode-as3mxml. The starting offset of a `MemberAccessExpressionNode` where `new ChainTest("foo", "bar")` is the left side and `chain(1, 2)` is the right side is not being set properly. It's incorrectly starting that the `.` character in between the two sides.

**Comment by joshtynjala** [comment]
Fixed in apache/royale-compiler@7fd59ea412090d6a53e0b935ed6f6d68e69f7df2. vscode-as3mxml needs to wait until the next Apache Royale release to include this fix, though.
